packages/react/src/textarea/Textarea.tsx

```diff
@@ -0,0 +1,77 @@
+import clsx from "clsx";
+import { type ComponentPropsWithRef, type ReactNode, forwardRef } from "react";
+import TextareaAutosize from "react-textarea-autosize";
+
+import type { ExtendProps } from "../utils";
+
+import { Box } from "../box";
+import * as styles from "./Textarea.css";
+
+type TextareaProps = ExtendProps<
+  ComponentPropsWithRef<"textarea">,
+  ComponentPropsWithRef<typeof Box>,
+  {
+    autosize?: boolean;
+    bottomSection?: ReactNode;
+    defaultValue?: string;
+    isDisabled?: boolean;
+    isInvalid?: boolean;
```

Comment:
let's rename `isDisabled` to `disabled`

and rename `isInvalid` to `error`

Reply:
addressed